easybuild/framework/easyconfig/tools.py

```diff
@@ -255,18 +255,32 @@ def get_paths_for(subdir=EASYCONFIGS_PKG_SUBDIR, robot_path=None):
     path_list.extend(sys.path)
 
     # figure out installation prefix, e.g. distutils install path for easyconfigs
-    eb_path = which('eb')
+
+    # prefer using path specified in $EB_SCRIPT_PATH (if defined), which is set by 'eb' wrapper script
+    eb_path = os.getenv('EB_SCRIPT_PATH')
+    if eb_path is None:
+        # try to determine location of 'eb' script via $PATH, as fallback mechanism
+        eb_path = which('eb')
+
```

Comment:
Why not do this the same way as below? getenv or which?

Reply:
Well, I could, doesn't matter.

But this way, we can probably log how we obtained the full path to `eb` (which I'm not doing yet here, but I should)

Reply:
okidoki